crafting_pieces.xml

```diff
@@ -13,12 +13,14 @@
     </Materials>
   </CraftingPiece>
   <CraftingPiece id="crpg_knights_fall_mace_t3_handle" name="{=qZYeu55z}Long Iron Bottomed Mace Handle" tier="3" piece_type="Handle" mesh="mace_handle_12" length="78" weight="0.75" item_holster_pos_shift="0,0,0.05">
+    <BuildData piece_offset="15.5" />
     <Materials>
       <Material id="Wood" count="1" />

```

Comment:
Why? Has this been tested?

Reply:
@namidaka these were in comparison to other values for 1h maces, there are a handful that are held in the middle of the weapon, rather than the base. Were there other values I should use to fix that issue?

Reply:
Piece offset depends on length. It needs to be at max slighty lower than hal of length. We'll see in beta testing